python/ctsm/crop_calendars/check_rx_obeyed.py

```diff
@@ -104,6 +104,27 @@ def get_extreme_info(diff_array, rx_array, mxn, dims, gs_da, patches1d_lon, patc
     return round(themxn, 3), round(this_lon, 3), round(this_lat, 3), this_gs, round(this_rx)
 
 
+def summarize_results(which_ds, output_var, verbose, all_ok, gdd_tolerance, diffs_eg_txt):
+    """
+    Summarize results
+    """
+    bad = True
+    if all_ok == 2:
+        bad = False
+        print(f"✅ {which_ds}: Prescribed {output_var} always obeyed")
+    elif all_ok == 1:
+        bad = False
+        # print(f"🟨 {which_ds}: Prescribed {output_var} *not* always obeyed, but acceptable:")
```

Comment:
Why is this line and the next commented out?

Reply:
Those lines provide a lot of extra verbosity, but they don't need to be on by default. It's really more of a development thing.

Reply:
I'd like to see this either: removed for readability, adding a comment to explain the purpose, or hook them up to a verbose or debug option so it's invoked then.

Reply:
Resolved with 400c4590b.